doc/languages-frameworks/perl.section.md

```diff
@@ -139,9 +142,10 @@ This program takes a Perl module name, looks it up on CPAN, fetches and unpacks
 ```ShellSession
 $ nix-generate-from-cpan XML::Simple
   XMLSimple = buildPerlPackage rec {
-    name = "XML-Simple-2.22";
+    pname = "XML-Simple";
+    version = "2.22";
     src = fetchurl {
-      url = "mirror://cpan/authors/id/G/GR/GRANTM/${name}.tar.gz";
+      url = "mirror://cpan/authors/id/G/GR/GRANTM/XML-Simple-2.22.tar.gz";
```

Comment:
```suggestion
      url = "mirror://cpan/authors/id/G/GR/GRANTM/${pname}-${version}.tar.gz";
```
?

Reply:
pname shouldn't be substituted here. Perl is sometimes very inconsistent with it.